README.md

```diff
@@ -20,16 +18,64 @@
+
+You have also been provided with a method for turning strings into colors. For example, you can call `"red".color` to get a red color. `"blue".color`, `"yellow".color`, and `"green".color` are also allowed. This should help you easily create `UIColor` objects and use them in the existing methods.
+
+You can also call `color` on segmented control objects. For example, given a segmented control called `sender` (such as the one passed to `colorSelected(_:`), you can call `sender.color` to get the color associated with it.
+
+Finally, there are three method stubs that have `TODO` markers indicating code that _you_ have to write: `viewDidLoad()`, `changeColor(to:)`, and `colorSelected(_:)`.
+
+Now, on to your tasks.
+
+### Your Tasks
 
-*  Considering this is the first lab encompassing what it is they student learned up to this point in this unit, I think we can make this one slightly easier considering there are scheduled to be two more labs after this one.  
-* Maybe it's something like this [lab](https://github.com/learn-co-curriculum/swift-viewLifeCycle-lab) but not as difficult.  
-* Free to explore ideas or fun projects the student can take on at this point.
+If you complete the following steps, you should end up with a properly working app:
 
+1. Hook up the `lightBulb` IB outlet in `ViewController.swift` with the light bulb view in `Main.storyboard`.
+2. Add a segmented control underneath the lightbulb.
+	* By default, a segmented control only has 2 buttons. Yours needs to have 4. Do you know how you can change that? (If you guessed "in the Attributes Inspector", you'd be correct!)
+	* The individual parts of a segmented control also have default names. Try to figure out on your own how you can change those names. (Hint: Look for a section in the Attributes Inspector called **Segment**. It is a popup menu that should list all the parts of your segmented control. You'll want to change an attribute called **Title** for each segment part.)
+3. Hook up the segmented control to the `colorSelected(_:)` IB outlet in `ViewController.swift`.
+4. Modify the `colorSelected(_:)` method where shown. It should change the background color the `lightBulb` view to the chosen color.
+	* Doing this might be a bit tricky. Notice that there is a `print` statement above the part of this method that you need to modify. That `print` statement is a hint as to how you can figure out which segment the user selected. Remember: You can get help using Xcode's documentation.
+	* The selected part of the segmented control will be returned as an `Int`. Remember: You have already been provided with a method that can turn an `Int` into a color!
+5. Modify the `changeColor(_:)` method to change the background color of the `lightBulb` view.
+6. Finally, modify `viewDidLoad()` to set the initial color of the light bulb to red.
+	* Remember: You have two methods to help you. One is `colorFromString(_:)`, which can take a string like "red" or "blue" and return a color.
```

Comment:
I don't see this `colorFromString(_:)` method. I'm down for it to be included, if you do wind up leaving it out I think then we should show them how to work with your extension with an example like this:

```swift
view.backgroundColor = "blue".color
```